model/src/main/java/org/cbioportal/model/CNA.java

```diff
@@ -0,0 +1,39 @@
+package org.cbioportal.model;
```

Comment:
Added as a step in the direction towards consolidating on a CNA enum shared by both importer and website processes.  When core code is cleaned/decomposed we should update to use this enum.